src/Object/Post.php

```diff
@@ -497,6 +503,7 @@ public function getTemplateData(array $conv_responses, $thread_level = 1)
 			'uriid'           => $item['uri-id'],
 			'return'          => (DI::args()->getCommand()) ? bin2hex(DI::args()->getCommand()) : '',
 			'direction'       => $direction,
+			'reshared'        => $reshared,
```

Comment:
The following code makes the 6 new lines above obsolete.
```suggestion
			'reshared'        => $item['reshared'] ?? '',
```